src/logic/nfts/collections.ts

```diff
@@ -118,45 +133,94 @@ export function getCollectionsOrderBy(
+  switch (fragment.itemType) {
+    case FragmentItemType.WEARABLE_V1:
+    case FragmentItemType.WEARABLE_V2:
+    case FragmentItemType.SMART_WEARABLE_V1: {
+      const result: NFTResult = {
+        nft: {
+          id: getId(fragment.contractAddress, fragment.tokenId),
+          tokenId: fragment.tokenId,
+          contractAddress: fragment.contractAddress,
+          activeOrderId:
+            fragment.activeOrder && !isExpired(fragment.activeOrder.expiresAt)
+              ? fragment.activeOrder.id
+              : null,
+          owner: fragment.owner.address.toLowerCase(),
+          name: fragment.metadata.wearable!.name,
+          image: fragment.image,
+          url: `/contracts/${fragment.contractAddress}/tokens/${fragment.tokenId}`,
+          data: {
+            wearable: {
+              bodyShapes: fragment.metadata.wearable!.bodyShapes,
+              category: fragment.metadata.wearable!.category,
+              description: fragment.metadata.wearable!.description,
+              rarity: fragment.metadata.wearable!.rarity,
+              isSmart: fragment.itemType === FragmentItemType.SMART_WEARABLE_V1,
+            },
+          },
+          issuedId: fragment.issuedId,
+          itemId: fragment.itemBlockchainId,
+          category: NFTCategory.WEARABLE,
+          network: Network.MATIC,
+          chainId: getCollectionsChainId(),
+          createdAt: +fragment.createdAt * 1000,
+          updatedAt: +fragment.updatedAt * 1000,
+          //@ts-ignore
+          soldAt: +fragment.soldAt * 1000,
         },
-      },
-      issuedId: fragment.issuedId,
-      itemId: fragment.itemBlockchainId,
-      category: NFTCategory.WEARABLE,
-      network: Network.MATIC,
-      chainId: getCollectionsChainId(),
-      createdAt: +fragment.createdAt * 1000,
-      updatedAt: +fragment.updatedAt * 1000,
-      //@ts-ignore
-      soldAt: +fragment.soldAt * 1000,
-    },
-    order:
-      fragment.activeOrder && !isExpired(fragment.activeOrder.expiresAt)
-        ? fromCollectionsOrderFragment(fragment.activeOrder)
-        : null,
-  }
+        order:
+          fragment.activeOrder && !isExpired(fragment.activeOrder.expiresAt)
+            ? fromCollectionsOrderFragment(fragment.activeOrder)
+            : null,
+      }
+
+      return result
+    }
+    case FragmentItemType.EMOTE_V1: {
+      const result: NFTResult = {
+        nft: {
+          id: getId(fragment.contractAddress, fragment.tokenId),
+          tokenId: fragment.tokenId,
+          contractAddress: fragment.contractAddress,
+          activeOrderId:
+            fragment.activeOrder && !isExpired(fragment.activeOrder.expiresAt)
+              ? fragment.activeOrder.id
+              : null,
+          owner: fragment.owner.address.toLowerCase(),
+          name: fragment.metadata.emote!.name,
+          image: fragment.image,
+          url: `/contracts/${fragment.contractAddress}/tokens/${fragment.tokenId}`,
+          data: {
+            emote: {
```

Comment:
If emote is the only thing that changes from a wearable, maybe wrap the rest in a function a provide the data

Reply:
Or whatever seems fit. Just to avoid repeating the chunks of code

Reply:
done 👍